.github/workflows/ci.yml

```diff
@@ -3,10 +3,10 @@ on:
     branches:
       - main
       - "*_dev"
-  pull_request:
-  merge_group:
-  schedule:
-    - cron: '0 18 * * *'
+  #pull_request:
+  # merge_group:
+  #schedule:
+  #  - cron: '0 18 * * *'
```

Comment:
Is there a reason to disable these? Leftover from testing?

Reply:
This is just a testing artifact, yes. I didn't want to waste CI resources when I was pushing to this branch a lot during development. I'll make sure to undo this before merging.